src/Events/Integrations/Plugins/Event_Tickets/Emails/Email/RSVP.php

```diff
@@ -82,12 +83,12 @@ public function include_settings( $settings ): array {
 	 * @since TBD
 	 *
 	 * @param array<string,string> $attachments  The attachments for the Tickets Emails.
-	 * @param string               $email_id     The email ID.
-	 * @param Email_Abstract       $email_class  The email class.
+	 * @param Dispatcher           $dispatcher  The email class.
```

Comment:
```suggestion
	 * @param Dispatcher           $dispatcher  The Email dispatcher object.
```

Should this be `The email class` or the above?